sdk/forms/src/test/java/com/klaviyo/forms/bridge/KlaviyoJsBridgeTest.kt

```diff
@@ -29,11 +31,38 @@ class KlaviyoJsBridgeTest : BaseTest() {
     fun `compileJson returns correct JSON for supported functions`() {
         val expectedJson = KlaviyoJsBridge().handshake.compileJson()
         val actualJson = """
-           [{"type":"profileMutation","version":1},{"type":"lifecycleEvent","version":1},{"type":"closeForm","version":1}]
+           [{"type":"profileMutation","version":1},{"type":"lifecycleEvent","version":1},{"type":"closeForm","version":1},{"type":"profileEvent","version":1}]
         """.trimIndent()
         assertEquals(actualJson, expectedJson)
     }
 
+    @Test
+    fun `profileEvent calls JS evaluator with correct JS`() {
+        val testEvent = Event(
+            metric = "Fate Sealed",
+            properties = mapOf(
+                EventKey.CUSTOM("name") to "Anna Karenina",
+                EventKey.CUSTOM("action") to "Adultery",
+                EventKey.CUSTOM("location") to "Saint Petersburg"
+            )
+        )
+
+        every { jsEvaluator.evaluateJavascript(any(), any()) } answers {
+            secondArg<(Boolean) -> Unit>().invoke(true)
+        }
+
+        bridge.profileEvent(testEvent)
+
+        verify {
+            jsEvaluator.evaluateJavascript(
+                eq(
+                    """window.profileMutation("Fate Sealed","{name=Anna Karenina, action=Adultery, location=Saint Petersburg}")"""
```

Comment:
this isn't valid json

Reply:
Oh and wrong JS function. There's one in [here](https://github.com/klaviyo/klaviyo-android-sdk/blob/186bc387d0a3ebf848fc279a587a30a2c62009ab/sdk/forms/src/main/assets/klaviyo-js-bridge.js#L53) already, `analyticsEvent` although I think the name needs fixing, I think we updated that to `profileEvent` to avoid ambiguity with `track-analytics` aggregate events.